src/select.coffee

```diff
@@ -209,6 +209,13 @@ Ember.AddeparMixins.ResizeHandlerMixin, Ember.Widgets.KeyboardHelper,
       tagName: 'span'
       labelPath: Ember.computed.alias 'controller.optionLabelPath'
       context: Ember.computed.alias 'controller.selection'
+      ###*
+      * This is to override the click event in the SelectOptionView
+      * so that we can make sure only when we click on a new item in the list,
+      * the userDidSelect action will be fired, not when we open the dropdown.
```

Comment:
I would add some clarification here that this view for the selected item is an extension of the view used to display each option in the popover list, and we don't want the click handler for the options in the popover list to be triggered when you click on the selected item. It's not really about the `userDidSelect` action. That just happens to be one of the effects of the click handler